restcomm/restcomm.mscontrol.jsr309/src/main/java/org/restcomm/connect/mscontrol/jsr309/Jsr309CallController.java

```diff
@@ -122,7 +122,7 @@
     private final State failed;
 
     // JSR-309 runtime stuff
-    private static final String[] CODEC_POLICY_AUDIO = new String[] { "audio" };
+    private static final String[] CODEC_POLICY_AUDIO = new String[] { "audio", "video" };
```

Comment:
If we always enable audio and video, the declaring codec policy is not needed.
This should probably be configurable.

@ghjansen have you tested all possibly scenarios to understand if having video always enabled may cause problems? Interop issues with PSTN or audio-only systems come to mind. Be careful here.